src/components/Synthetics/AllowedSwapSlippageInputRowImpl/AllowedSwapSlippageInputRowImpl.tsx

```diff
@@ -0,0 +1,119 @@
+import { Trans, t } from "@lingui/macro";
+import { memo, useCallback, useMemo } from "react";
+
+import { HIGH_ALLOWED_SWAP_SLIPPAGE_BPS } from "config/factors";
+import { formatPercentage } from "lib/numbers";
+
+import ExchangeInfoRow from "components/Exchange/ExchangeInfoRow";
+import PercentageInput from "components/PercentageInput/PercentageInput";
+
+import "../AcceptablePriceImpactInputRow/AcceptablePriceImpactInputRow.scss";
+
+type Props = {
+  allowedSwapSlippageBps?: bigint;
+  recommendedAllowedSwapSlippageBps?: bigint;
+  initialSwapImpactFeeBps?: bigint;
+  setAllowedSwapSlippageBps: (value: bigint) => void;
+  totalSwapImpactBps: bigint;
+  notAvailable?: boolean;
```

Comment:
probably rename it to  “disabled”?  would be a more familiar name

Reply:
this inherited from AcceptableSwapImpact component, and I think naming is ok cuz it will render N/A, it's differs from just disabled state